shell/platform/darwin/macos/framework/Source/FlutterViewProvider.h

```diff
@@ -15,11 +15,18 @@ extern const uint64_t kFlutterDefaultViewId;
  */
 @protocol FlutterViewProvider
 
+/**
+ * Get the FlutterView for the default view.
+ *
+ * Returns nil if it doesn't exist.
+ */
+- (nullable FlutterView*)getDefaultView;
```

Comment:
Do we have to add this? Ideally, we should no longer make the assumption that there is a default view. If we have this, it bags the question: What is the default view and why is it special?

Reply:
Agreed - I'd avoid this. It feels like adding new API for a legacy concept. Since this isn't public API, it's probably actually advantageous to force the longhand version of looking it up by default ID where needed, just to make the code more explicit and to avoid people accidentally adding more use of single-view assumptions/conveniences.

If, for whatever reason, we somehow absolutely can't live without this method, it should be called `defaultView` not `getDefaultView`. Getter's don't take a leading 'get' in Obj-C.